tensilelite/Tensile/Source/client/main.cpp

```diff
@@ -135,7 +135,8 @@ namespace Tensile
                 ("init-scaleB",              po::value<InitMode>()->default_value(InitMode::Two), "Initialization for scaleB")
                 ("init-scaleC",              po::value<InitMode>()->default_value(InitMode::Two), "Initialization for scaleC")
                 ("init-scaleD",              po::value<InitMode>()->default_value(InitMode::Two), "Initialization for scaleD")
-                ("init-scaleAlphaVec",           po::value<InitMode>()->default_value(InitMode::One), "Initialization for scaleAlphaVec")
+                ("init-scaleAlphaVec",       po::value<InitMode>()->default_value(InitMode::One), "Initialization for scaleAlphaVec")
+                ("init-Synchronizer",        po::value<InitMode>()->default_value(InitMode::Zero), "Initialization for Synchronizer")
```

Comment:
Should not add this argument. synchronizer is solution dependent but not problem type dependent.

Reply:
removed

Reply:
removed